src/RateLimiting/src/ConcurrencyLimiterOptions.cs

```diff
@@ -5,22 +5,49 @@
 
 namespace System.Threading.RateLimiting
 {
-    sealed public class ConcurrencyLimiterOptions
+    /// <summary>
+    /// Options to control the behavior of a <see cref="ConcurrencyLimiter"/>.
+    /// </summary>
+    public sealed class ConcurrencyLimiterOptions
     {
+        /// <summary>
+        /// Initializes the <see cref="ConcurrencyLimiterOptions"/>.
+        /// </summary>
+        /// <param name="permitLimit"></param>
+        /// <param name="queueProcessingOrder"></param>
+        /// <param name="queueLimit"></param>
+        /// <exception cref="ArgumentOutOfRangeException">When <paramref name="permitLimit"/> or <paramref name="queueLimit"/> are less than 0.</exception>
         public ConcurrencyLimiterOptions(int permitLimit, QueueProcessingOrder queueProcessingOrder, int queueLimit)
         {
+            if (permitLimit < 0)
+            {
+                throw new ArgumentOutOfRangeException(nameof(permitLimit));
+            }
+            if (queueLimit < 0)
+            {
+                throw new ArgumentOutOfRangeException(nameof(queueLimit));
+            }
             PermitLimit = permitLimit;
             QueueProcessingOrder = queueProcessingOrder;
             QueueLimit = queueLimit;
         }
 
-        // Maximum number of permits allowed to be leased
+        /// <summary>
+        /// Maximum number of permits allowed to be leased.
+        /// </summary>
         public int PermitLimit { get; }
 
-        // Behaviour of WaitAsync when not enough resources can be leased
-        public QueueProcessingOrder QueueProcessingOrder { get; }
+        /// <summary>
+        /// Determines the behaviour of <see cref="RateLimiter.WaitAsync"/> when not enough resources can be leased.
+        /// </summary>
+        /// <value>
+        /// <see cref="QueueProcessingOrder.OldestFirst"/>
```

Comment:
I don't think `<value>` is intended to indicate just the default value unless you say that it is: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#value

```suggestion
        /// <see cref="QueueProcessingOrder.OldestFirst"/> by default.
```